sites/skeleton.dev/src/lib/components/DocsIcon/icons.ts

```diff
@@ -3,8 +3,18 @@
     License - https://fontawesome.com/license (Commercial License)
     Copyright 2022 Fonticons, Inc
 */
+export type Icon = {
+    viewBox?: string;
+    path: string;
+};
+
+export type Icons = {
```

Comment:
@jwatts777 please add `[key: string]: Icon;` here so we can index `Icons` using strings. this solves 2 of svelte-check errors.